lib/matplotlib/colorbar.py

```diff
@@ -417,9 +415,9 @@ def __init__(self, ax, *, cmap=None,
             ['uniform', 'proportional'], spacing=spacing)
 
         # wrap the axes so that it can be positioned as an inset axes:
-        ax = ColorbarAxes(ax, userax=userax)
-        self.ax = ax
-        ax.set(navigate=False)
+        super().__init__(ax, userax=userax)
+        self.ax = self
```

Comment:
I'd suggest making this a property instead: it would make room to add a warning when this "attribute" is deprecated.

Reply:
That's right. I guess you could see this as premature :)